### DISCUSSION TITLE
Does this tool will be deprecated soon?

### DISCUSSION THREAD

**Discussion by xheisenbugx** [sturdy-dev/sturdy#1338] [General]
I would like to start using this tool for my current git workflow but then I realized that there is a banner in the webpage of the project saying that sturdy is not in active development, doest that mean that the tool will be deprecated or it meas that the devs will work on it later?

**Comment by zegl** [comment]
Sturdy is still working and we're maintaining it and making sure that it keeps running and working.

We're not dedicating as much resources to Sturdy as we used to, but this can change for the better again in the future. If you like it, you should use it!
